ang/test/civicase-base/directives/inline-datepicker.directive.spec.js

```diff
@@ -0,0 +1,130 @@
+/* eslint-env jasmine */
+
+(($) => {
+  describe('civicaseInlineDatepicker', () => {
+    const NG_INVALID_CLASS = 'ng-invalid';
+    let $compile, $rootScope, $scope, element, originalDatepickerFunction,
+      removeDatePickerHrefs;
+
+    beforeEach(module('civicase-base', 'civicase.data'));
+
+    beforeEach(inject((_$compile_, _$rootScope_, _removeDatePickerHrefs_) => {
+      $compile = _$compile_;
+      $rootScope = _$rootScope_;
+      removeDatePickerHrefs = _removeDatePickerHrefs_;
+      $scope = $rootScope.$new();
+      originalDatepickerFunction = $.fn.datepicker;
+      $.fn.datepicker = jasmine.createSpy('datepicker');
+      moment.suppressDeprecationWarnings = true;
+    }));
+
+    afterEach(() => {
+      $.fn.datepicker = originalDatepickerFunction;
+      moment.suppressDeprecationWarnings = false;
+    });
+
+    describe('when the directive is initialised', () => {
+      beforeEach(() => {
+        initDirective();
+      });
+
+      it('sets the element as a datepicker input element', () => {
+        expect($.fn.datepicker).toHaveBeenCalled();
+      });
+
+      it('removes the HREF attributes from the datepicker links', () => {
```

Comment:
Lets put more of a generic case name. For example: "does not change the site url wrongly when selecting date".